test/coverage.js

```diff
@@ -232,4 +233,32 @@ describe('Coverage', () => {
             done();
         });
     });
+
+    describe('Clear require cache', () => {
+
+        const cacheBackup = require.cache; // backup require cache
+
+        lab.before((done) => {
+
+            require.cache = Module._cache = {}; // clear require cache
+            done()
+        })
+
+        lab.after((done) => {
+
+            require.cache = Module._cache = cacheBackup; // restore require cache
+            done()
+        })
+
+        it('does not reset file coverage', function (done) {
```

Comment:
If you remove the lab.before and lab.after and put the code that changes the cache directly into the test, does it work?